docs/src/css/custom.css

```
@@ -1,71 +1,220 @@
+  --ifm-color-warning: var(--obot-yellow);
+  --ifm-color-warning-dark: #d9ab00;
+  --ifm-color-warning-contrast-background: #fff9e0;
+  --ifm-color-warning-contrast-foreground: #4d3a00;
+  /* caution / danger -> brand red */
+  --ifm-color-danger: var(--obot-red);
+  --ifm-color-danger-dark: #e01b20;
+  --ifm-color-danger-contrast-background: #ffecec;
+  --ifm-color-danger-contrast-foreground: #5c0f11;
+
+  /* ---- Typography ---- */
   --ifm-font-family-base: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)